glotaran/builtin/models/kinetic_spectrum/spectral_penalties.py

```diff
@@ -100,6 +100,10 @@ def apply_spectral_penalties(
 
 
 def _get_idx_from_interval(interval, axis):
```

Comment:
Could you also add the types to the function signature? 😄 
I guess it should be
```python
def _get_idx_from_interval(interval: Tuple[float, float], axis: Deque[float]) ->Tuple[int, int]:
```

Reply:
Or to be more general 
```python
def _get_idx_from_interval(interval: Tuple[float, float], axis: Sequence[float]) ->Tuple[int, int]:
```